tools/maxquant/maxquant.xml

```diff
@@ -92,24 +93,24 @@
                type="data" label="FASTA files"
                help="Specify one or more FASTA databases. Parse rules are hard coded:
                      '&gt;([^\s]*)' for the identifier, '&gt;(.*)' for the description." />
-        <conditional name="fractions">
-            <param name="set_fractions" type="boolean"
-                   label="specify fractions?" checked="false"/>
-            <when value="true">
-                <param format="tabular" name="template" type="data"
-                       label="Specify an experimental design template. Currently 
-                              four columns are supported: Name, Fraction, Experiment and 
-                              PTM. Values have to be tab-separated." />
-            </when>
-            <when value="false" />
-        </conditional>
                 
         <conditional name="mode">
             <param name="choose_mode" type="select" label="choose mode">
                 <option value="interactive">specify search parameters</option>
                 <option value="mqpar">specify mqpar.xml</option>
             </param>
             <when value="interactive">
+                <conditional name="fractions">
+                    <param name="set_fractions" type="boolean"
+                           label="specify fractions?" checked="false"/>
+                    <when value="true">
+                        <param format="tabular" name="template" type="data"
```

Comment:
@dglaetzer you can remove the entire conditional here and make the data-param `optional="True"` that serves the same purpose.